event-internal.h

```diff
@@ -368,7 +368,7 @@ struct event_config {
 };
 
 /* Internal use only: Functions that might be missing from <sys/queue.h> */
-#if defined(EVENT__HAVE_SYS_QUEUE_H) && !defined(EVENT__HAVE_TAILQFOREACH)
```

Comment:
I presumed this was because you didn't want to:

1) miss sys/queue.h
2) define a few odd functions, leaving others undefined.

I will be happy to remove the EVENT__HAVE_SYS_QUEUE_H guard as well.